### PR TITLE
Merchant automation sematics

### PR DESCRIPTION
**Pull request by alphapokedex**

### CODE REVIEW

File: packages/regal/.fvm/fvm_config.json [hunk 1]
```diff
@@ -1,4 +1,3 @@
 {
-  "flutterSdkVersion": "3.10.6",
-  "flavors": {}
+  "flutterSdkVersion": "stable"
```
Comment (vamsi-kleargroup) [inline packages/regal/.fvm/fvm_config.json:2]: stable will mean different depending on dev's laptop config

Comment (alphapokedex) [reply packages/regal/.fvm/fvm_config.json:2]: i'll keep it to 3.13.9 as minimum then

Comment (alphapokedex) [reply packages/regal/.fvm/fvm_config.json:2]: better yet i'll remove fvm config from git history @vamsi-kleargroup bro?

Comment (vamsi-kleargroup) [reply packages/regal/.fvm/fvm_config.json:2]: nah..its better to have..everyone can update if we updated it

**Review by amanxnanda — APPROVED** [review]

**Review by yasht01 — APPROVED** [review]

**Review by vamsi-kleargroup — APPROVED** [review]

**Pull request merged**